docs/releases/1.8.txt

```diff
@@ -1131,6 +1131,11 @@ Miscellaneous
   has been removed by a migration and replaced by a property. That means it's
   not possible to query or filter a ``ContentType`` by this field any longer.
 
+* :djadmin:`migrate` now accepts the :djadminopt:`--fake-initial` option to
+  allow faking initial migrations. In 1.7 initial migrations were automatically
```

Comment:
"to enable automatic faking of initial migrations." The automatic part is key; "allowing" makes it sound like you can't fake migrations otherwise, which of course you can with `--fake`